open-company/open-company-web
# Early code cleanup

card: https://trello.com/c/ZyAv8cYx/187-ui-earlier-code-cleanup

Most of the changes where suggested by kibit that now run also in CI.
Test:
- compile and run the dev env
- check that the site works
- try to make some changes to different companies just to make sure it works
- check travis
